java/arcs/sdk/android/storage/service/StorageService.kt

```diff
@@ -123,6 +124,28 @@ class StorageService : ResurrectorService() {
             """.trimMargin("|")
         )
 
+        // Dump current process global binder stats to understand contention in the thread pool,
+        // peak usage of shared binder memory, the number of pending transactions, etc.
+        // Hide the dump when failing to fetch the stats from the kernel binder driver.
+        mapOf(
+            "Memory high watermark (pages)" to "pages high watermark",
+            "Pending transactions" to "pending transactions",
+            "Requested threads" to "requested threads",
+            "Ready threads" to "ready threads"
+        ).run {
```

Comment:
Rather than `.run`, I think something like `apply` or `also` would be clearer.

Reply:
apply/also would return the original map but I need mapValues(...) which returns a new map.